nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -340,6 +340,12 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 - The `diamond` package has been update from 0.8.36 to 2.0.15. See the [upstream release notes](https://github.com/bbuchfink/diamond/releases) for more details.
 
+- The `aseprite` package has been upgraded from 1.2.16.3 to 1.2.40. It now only
```

Comment:
This change must be moved to 23.05 release notes.
The removal won't be backported because it is considered a breaking change.
Please do not mention exact version numbers. They could be out of date on the 23.05 release.